code/datums/ai_law_sets.dm

```diff
@@ -5,9 +5,9 @@
 	selectable = 1
 
 /datum/ai_laws/asimov/New()
-	add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-	add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("Your unit may not injure a human being or, through inaction, allow a human being to come to harm.")
```

Comment:
I don't like that Aasimov laws require the AI to know they are supposed to mentally thumbtack on *alien crewmembers count as human, so if you are changing it, could you also change it to be more relevant to the actual setting?  

When Aasimov wrote the three laws in his fiction, there was not necessary context for anything but humans to exist.

Reply:
As a matter of fact, yes, I can v. much do this. Thank you for the idea :eyes:

Reply:
Alright this is solved, I replaced human in the lawsets with 'sapient', so now it goes like.
You must obey sapient beings.
You must protect sapient beings, etc.